test/Microsoft.AspNetCore.Server.KestrelTests/LifetimeNotImplemented.cs

```diff
@@ -35,7 +35,6 @@ public CancellationToken ApplicationStopping
 
         public void StopApplication()
         {
-            throw new NotImplementedException();
```

Comment:
Whoa. Don't remove this. This should never be called, and this is used in all of our tests. If it is being called, I wan't to know about it.